### ISSUE TITLE
Toggle Click-through issues

### ISSUE THREAD

**Issue by megaroeny** [getflourish/Sketch-Mate#32]
When using the shortcut to toggle click-through, it also toggles all parent groups too. 😞

**Comment by listao** [comment]
Agreed. Issue the other way around as well. Toggling parent would set to all children as well...I would be nice if we could limit the toggle to the selected layer(s) only.

**Comment by ryanclaytoncodes** [comment]
Any update on this?
